### PR TITLE
Updating sponsorship page to include tiers

### PR DESCRIPTION
**Pull request by socksy**
I've called them Platinum, Gold, and Silver — given these are common names for funding tiers. Most of the text has been kept intact, with some rewording, and removing of various stuff that's no longer relevant. Donations types have been made more explicit, hopefully highlighting the fact that invoices are not given for the Travis Foundation donations. 

Feedback and corrections more than welcome.

### CODE REVIEW

File: sponsor_us.md [hunk 1]
```diff
@@ -17,40 +17,46 @@ By supporting us you help to make the world a little better. Your company also
...
-They receive a shoutout at the start and end of the event, and receive a warm
-"thank you" on our social media account. We also list sponsors by name on our
-popular t-shirts that are handed out to teachers and attendees, and provide the
-opportunity to offer "swag" (stickers, note books, thumb drives, etc.) at the
-event.
+Your company has an option of different tiers of support. There's the direct
+sponsorship tiers, which are the most valuable form of support, and a generic
+small supporter tier. The direct sponsorship tiers involve directly paying our
+supplier with the order details we provide, and as such also mean you can pay
+with credit card will and receive an invoice.
```
Comment (vsmart) [inline sponsor_us.md]: typo: and will receive

**Comment by vsmart** [comment]
Thanks for doing this! 🎉

File: sponsor_us.md [hunk 2]
```diff
@@ -17,40 +17,46 @@ By supporting us you help to make the world a little better. Your company also
...
+If you've opted for a direct sponsorship tier, then your transaction will be with
+our suppliers directly. In this case, we will give you the exact
+link/message/order that we need, and you make the transaction with that
+company directly. This means normal business billing, with ability to pay by
+credit card, and get an invoice. However, due to the nature of the process,
+we can't predict the exact amount — it would depend on factors such as the
+the number of sign-ups.
 
-This is an example budget based on previous workshops:
+### Support us via donation
```
Comment (vsmart) [inline sponsor_us.md:54]: I would mention the 250 here again explicitly.

**Comment by Malwine** [comment]
:+1: Thanks @socksy !

I am wondering if we should set up a `sponsor-google-form` or link our `twitter handle` too. Because sometimes people are lazy with emails.
Or we give the mail link a class that looks like an action button.

I love that it is shorter and gives a better overview!

File: sponsor_us.md (hunk 2 above)
Comment (socksy) [inline sponsor_us.md:54]: Perhaps we would be willing to take higher amounts from donations? We only have three tiers after all, and with a presumption that Cognitect is probably taking one of those.

I think that mentioning the 250 again would still be warranted, but I'm unsure how then to phrase it.

Comment (vsmart) [inline sponsor_us.md:54]: Oh, I was more concerned about the lower boundary, not the upper one. Maybe something as simple as `Starting from 250` would do.

Comment (socksy) [inline sponsor_us.md:54]: genius

**Comment by plexus** [comment]
Thanks @socksy this is starting to look :100:

File: sponsor_us.md [hunk 3]
```diff
@@ -17,40 +17,47 @@ By supporting us you help to make the world a little better. Your company also
...
-* Small costs (tags, postage, pens): € 30
+Alternatively, we also take donations via the
+[Travis Foundation UG](http://foundation.travis-ci.org/), a registered
+not-for-profit company (a “Gemeinnützige Organisation” under German law). This 
+is how we can take the small supporter tier, for donations from 250€. The Travis 
+Foundation can issue a Donation Receipt for accounting and tax deduction. It
+does *not* issue invoices for donations.
 
-Get in touch today at Clojurebridge.Berlin (at) gmail.com.
+Get in touch today at <clojurebridge.berlin@gmail.com>.
```
Comment (vsmart) [inline sponsor_us.md:63]: one tiny last change, let's add the twitter handle @clojurebridgede here, like @Malwine suggested. 
After that let's merge it \o/

**Comment by Malwine** [comment]
I am going to merge this and do 2 small changes on master afterwards.

**Comment by Malwine** [comment]
Ow and @socksy ! Thanks again for your PR!
giphy4

**Pull request merged**